src/java/jdtls.ts

```diff
@@ -10,6 +10,10 @@ export namespace Jdtls {
         return commands.executeCommand(Commands.EXECUTE_WORKSPACE_COMMAND, Commands.JAVA_PROJECT_LIST, params);
     }
 
+    export function updateFilters(...params): Thenable<boolean> {
```

Comment:
From the implementation, it only supports one array parameter. No need to use ...params